ConnectorRhino/ConnectorRhino/ConnectorRhinoShared/UI/ConnectorBindingsRhino2.cs

```diff
@@ -443,13 +461,17 @@ public override async Task SendStream(StreamState state, ProgressViewModel progr
           {
             if (!converter.CanConvertToSpeckle(obj))
             {
-              Exceptions.Add(new Exception($"Objects of type ${obj.Geometry.ObjectType} are not supported"));
+              var exception = new Exception($"Objects of type ${obj.Geometry.ObjectType} are not supported");
+              Exceptions.Add(exception);
+              progress.Report.LogOperationError(exception);
```

Comment:
We should not log an error if an object is not supported, instead, just log the text "Skipping object of type because..."